### PR TITLE
Change to HTTP Manager - standard cookie spec

### PR DESCRIPTION
**Pull request by jadecarino**
Signed-off-by: Jade Carino <jade.carino@ibm.com>

### CODE REVIEW

**Comment by jadecarino** [comment]
Hi, we made a change to the HTTP Manager yesterday, as we were having problems with the cookie headers in the responses to some requests, so have now set the cookie spec to standard. I have ran the HTTP IVT against these changes and this hasn't caused any regressions in that. @Michael-Baylis @hobbit1983

File: galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/internal/HttpClientImpl.java [hunk 1]
```diff
@@ -508,21 +507,24 @@ public IHttpClient setAuthorisation(String username, String password, URI scope)
...
+    	requestBuilder.setCookieSpec(CookieSpecs.STANDARD);
         builder.setDefaultCredentialsProvider(credentialsProvider);
         builder.setDefaultHeaders(commonHeaders);
 
         if (timeout > 0) {
-            RequestConfig.Builder requestBuilder = RequestConfig.custom().setConnectTimeout(timeout)
-                    .setConnectionRequestTimeout(timeout).setSocketTimeout(timeout);
+            requestBuilder.setConnectTimeout(timeout)
+                          .setConnectionRequestTimeout(timeout).setSocketTimeout(timeout);
             builder.setDefaultRequestConfig(requestBuilder.build());
```
Comment (Michael-Baylis) [inline galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/internal/HttpClientImpl.java:520]: this is being done on line 527,  so doesn't need to be done here.

**Review by Michael-Baylis — CHANGES_REQUESTED** [review]
code duplication

File: galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/internal/HttpClientImpl.java (hunk 1 above)
Comment (jadecarino) [reply galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/internal/HttpClientImpl.java]: thanks for pointing that out, have removed this and kept on line 527

**Review by Michael-Baylis — APPROVED** [review]

**Pull request merged**